### DISCUSSION TITLE
There is a trojan in a file that I wanted to download but I'm new and I don't know how to report it

### DISCUSSION THREAD

**Discussion by SteyKer** [community/community#40786] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, I'm new to github and I only go here to download things occasionally, but today I came across a file that has a Trojan, more specifically a "Trojan:Win32/DllInject" but I don't know or it's too complicated for me to find one way to report the file. The URL or where the file is located is the following: https://github.com/R3nzTheCodeGOD/R3nzSkin/releases/tag/v2.3.4 (this can also be used https://github.com/R3nzTheCodeGOD/R3nzSkin ).Hope that helps.

**Comment by airtower-luna** [comment]
If a user is distributing malware, you can report that user. The documentation for how to do that is here: https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam

Make sure to mention the exact file that's the problem in your report.
